### PR TITLE
RavenDB-6945

### PR DESCRIPTION
**Pull request by IdanHaim**
- Client send to the server the requested client url in the cluster
- fix typo

### CODE REVIEW

**Comment by ml054** [comment]
@IdanHaim  why do we have to pass server url to getClusterTopology command?

File: src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs [hunk 1]
```diff
@@ -51,15 +51,17 @@ public Task GetClusterTopology()
...
                 if (topology.Members.Count == 0)
                 {
+                    var serverUrl = GetStringQueryString("url");
                     topology = new ClusterTopology(
                         Guid.NewGuid().ToString(),
                         null,
                         new Dictionary<string, string>
                         {
-                            ["A"] = HttpContext.Request.GetHostnameUrl()
+                            ["A"] = serverUrl
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs:64]: But if this is null, use the current one.

**Comment by ayende** [comment]
@ml054 The reason we pass the url to the server is that we might be routed through a proxy, so we need to report the url that the user has used to reach us.

File: src/Raven.Studio/typescript/commands/database/cluster/getClusterTopologyCommand.ts [hunk 2]
```diff
@@ -4,10 +4,18 @@ import clusterTopology = require("models/database/cluster/clusterTopology");
...
 class getClusterTopologyCommand extends commandBase {
 
+    constructor(private serverUrl?: string) {
+        super();
+    }
+
     execute(): JQueryPromise<clusterTopology> {
+        
+        const args = {
+            url: this.serverUrl
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/cluster/getClusterTopologyCommand.ts]: url: appUrl.baseUrl

**Comment by ml054** [comment]
Ok I fault I thought that the url we use to send request. However I talked with @IdanHaim and he is gonna change window.location.host with appUrl.baseUrl.

File: src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs (hunk 1 above)
Comment (IdanHaim) [reply src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs:64]: serverUrl can't be null it is required

**Pull request merged**